package.json

```diff
@@ -93,7 +93,7 @@
     "@mui/icons-material": "^5.16.14",
     "@mui/internal-babel-plugin-resolve-imports": "1.0.20",
     "@mui/internal-markdown": "^1.0.24",
-    "@mui/internal-test-utils": "^1.0.25",
+    "@mui/internal-test-utils": "https://pkg.csb.dev/mui/material-ui/commit/48c999b6/@mui/internal-test-utils",
```

Comment:
Temporary build from https://github.com/mui/material-ui/pull/45023 until new version is released.